solr/contrib/ltr/src/java/org/apache/solr/ltr/ranking/LTRFeatureLoggerTransformerFactory.java

```diff
@@ -50,9 +51,21 @@
  */
 public class LTRFeatureLoggerTransformerFactory extends TransformerFactory {
 
+  private String loggingModelName;
+  private String loggingModelClassName = LoggingModel.class.getCanonicalName();
+
+  public void setLoggingModelName(String loggingModelName) {
+    this.loggingModelName = loggingModelName;
```

Comment:
Doesn't this now force our clients to make a required config change to set a name?  The name was a hidden aspect that users didn't have to know about before.  We should at least support a default one if not provided, and if it clashes with one of their own model names they can override it now.
Should probably have a test that tries to load a model with a clashing name of the logging model's default.